common/main/java/com/couchbase/lite/AbstractReplicator.java

```diff
@@ -70,12 +66,9 @@
  * be notified of progress.
  */
 @SuppressWarnings({"PMD.GodClass", "PMD.CyclomaticComplexity", "PMD.TooManyMethods", "PMD.TooManyFields"})
-public abstract class AbstractReplicator extends NetworkReachabilityListener {
+public abstract class AbstractReplicator {
```

Comment:
I think NetworkReachabilityListener is still needed. Basically the platform still needs to listener to the reachability change when the replicator's status of OFFLINE and inform the C4Replicator that the host becomes reachable via c4repl_setHostReachable() API.

Reply:
I just saw CBL-760.